spec/derivative_services/default_derivative_service_spec.rb

```diff
@@ -60,8 +59,6 @@
     expect(derivative).to be_present
     derivative_file = Valkyrie::StorageAdapter.find_by(id: derivative.file_identifiers.first)
     expect(derivative_file.read).not_to be_blank
-    # Ensure only one transaction is used
-    expect(adapter.metadata_adapter.connection).to have_received(:transaction).exactly(1).times
```

Comment:
Why did we care about the number of transactions before? And why don't we care anymore?

Reply:
Thank you. The research here was fruitful - pushing an update that fixes this test to check for the buffer rather than the transaction, and links to https://github.com/pulibrary/figgy/issues/2188 (which is why this feature was added.)

This started to fail because failable transactions are now used to check if an invalid ID was given to postgres. 

https://github.com/samvera-labs/valkyrie-sequel/pull/41